WcaOnRails/app/controllers/relations_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+class RelationsController < ApplicationController
+  def index
+    render :relation
+  end
+
+  def relation
+    @wca_id1 = params[:wca_id1]
+    @wca_id2 = params[:wca_id2]
+    @selected_people = Person.where(wca_id: [@wca_id1, @wca_id2]).includes(:user)
+    if @selected_people.count != 2
+      flash[:danger] = I18n.t('relations.messages.invalid_wca_ids')
```

Comment:
Note to self: this will happen if you choose the same person twice. Not sure how this ends up feeling for a user.